website/docs/reference/global-configs.md

```diff
@@ -97,24 +97,17 @@ $ dbt --debug run
 
 ## Log Formatting
 
-The `LOG_FORMAT` config specifies how dbt's logs should be formatted. The value for this flag can be one of: `text`, `json,` or `default`. Use the `json` formatting value in conjunction with `DEBUG` flag to produce rich log information which can be piped into monitoring tools for analysis.
-
-When `json` log formatting is used, each log line will have the following JSON properties:
-- timestamp: when the log line was printed
-- message: the textual log message
-- channel: the source for the log (eg. `dbt`, or `some_module`)
-- level: an integer indicating the log level for the log line (10=info, 11=debug, ...)
-- levelname: a string representation of the log level
-- thread_name: the thread in which the log message was produced
-- process: the PID for the running dbt invocation which produced this log message
-- extra: a dictionary containing "extra" information about the log line. This contents of this dictionary vary based on the specific log message that is being emitted. This dictionary contains programmatically accessible information to contextualize the log message.
+The `LOG_FORMAT` config specifies how dbt's logs should be formatted. If the value of this config is `json`, dbt will output fully structured logs in JSON format; otherwise, it will output text-formatted logs that are sparser for the CLI and more detailed in `logs/dbt.log`.
+
+Use `json` formatting value in conjunction with the `DEBUG` config to produce rich log information which can be piped into monitoring tools for analysis.
+
+See [structured logging](events-logging#structured-logging) for more details.
 
 <File name='Usage'>
 
 ```text
 $ dbt --log-format json run
```

Comment:
Should we show the `DEBUG` config  mentioned above?

Reply:
Hm yeah. JSON formatting can be used with or without `DEBUG`. This is more like a protip than a specific usage requirement. I'll see if I can find a way to make it happen